lib/ci-helper.ts

```diff
@@ -662,7 +662,7 @@ export class CIHelper {
                     }
                 } catch (reason) {
                     throw new Error(`User ${
-                        accountName} is not a valid GitHub username: ` + (reason || ''));
+                        accountName} is not a valid GitHub username: ${reason}`);
```

Comment:
Oops. I guess this line is still longer than 80 columns.
```suggestion
                        accountName} is not a valid GitHub username: ${
                        reason}`);
```

Reply:
I have disabled the 80 char lint limit locally, because I can't properly read the code with all that weird wrapping applied...